cloudsql/sqlserver/database-sql/cloudsql.go

```diff
@@ -0,0 +1,302 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Sample database-sql demonstrates connection to a Cloud SQL instance from App Engine
+// standard. The application is a Golang version of the "Tabs vs Spaces" web
+// app presented at Cloud Next '19 as seen in this video:
+// https://www.youtube.com/watch?v=qVgzP3PsXFw&t=1833s
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"html/template"
+	"log"
+	"math"
+	"net/http"
+	"os"
+	"strconv"
+
+	_ "github.com/denisenkom/go-mssqldb"
+)
+
+// db is the global database connection pool.
+var db *sql.DB
```

Comment:
Optional: other apps have followed a pattern of having a `type app struct { ... }` that contains all of the state that would otherwise be global. Then, you create the `app` in `func main` and the handlers can be defined as methods of that type to get access to the fields. That way, you avoid having any global state.

Reply:
If you decide to do this, please update the MySQL so they don't drift too much.

Reply:
I've added the App struct and used its db property for global database connection, with the index handler as a method of the struct. @tbpg let me know if this looks OK for the general approach, and if so I'll update the MySQL sample to match.